### PR TITLE
Add jupyter notebooks for the examples

### PR DESCRIPTION
**Pull request by fwmeng88**
This PR is mainly for adding Jupyter notebooks for the examples that use Procrustes. 

- [x] Fix pylint error by disabling unbalanced-tuple-unpacking warning
- [x] Atom-atom mapping 
- [x] Rank and reordering
- [x] Protein structure alignment
- [x] Chiral center check
- [x] Add a badge for mybinder.org link which enables people to use jupyter notebook online

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -170,6 +170,8 @@ disable=
     # method.
     W0221,
     # unecessary "else" after "return" (R1705)
+    W0632,
```
Comment (FarnazH) [inline tox.ini:173]: I don't believe this needs to be disabled. There is an unbalance in the function return values and the variables used to unpack it, which needs to be fixed.

**Comment by FarnazH** [comment]
The PR https://github.com/theochem/procrustes/pull/8 fixes linter's `W0632`, so 58c50ec should be removed now. Alos, Paul's PR is merged so, c05894c should be removed now. Please make these changes and rebase your commits.

**Comment by fwmeng88** [comment]
> The PR #8 fixes linter's `W0632`, so [58c50ec](https://github.com/theochem/procrustes/commit/58c50ece691661efc21a5c115d43c507f686aeae) should be removed now. Also, Paul's PR is merged so, [c05894c](https://github.com/theochem/procrustes/commit/c05894cff8f7d44e44286dd686c010a85fa952b3) should be removed now. Please make these changes and rebase your commits.
Thanks, Farnaz! I will do it shortly.

File: tox.ini (hunk 1 above)
Comment (fwmeng88) [reply tox.ini]: I have rebased by dropped the committee. Thanks for pointing it out.

Comment (fwmeng88) [reply tox.ini]: Problem fixed in bdc3cd1. Thanks for the help.

**Pull request merged**